src/test/java/teammates/test/cases/testdriver/HtmlHelperTest.java

```diff
@@ -135,4 +135,20 @@ public void testConvertToStandardHtmlPart() throws Exception {
         
         //other cases are tested in testComparison
     }
+    
+    @Test
+    public void testReplacement() throws IOException {
```

Comment:
No, you misunderstood it entirely. Here what is being tested is the process of HTML verification, i.e the replacement algorithms. Copy-pasting from the gist I wrote:

- Expected string, from test file → inject test properties → convert to standard HTML
- Actual string, from web page → suppress variations of injected values, replace non-injected values with wildcards/placeholders → convert to standard HTML

Now take a look at how the `expected` and `actual` are processed in the test and you'll see that they match the above steps. That's exactly what's being tested. This test has no knowledge of this thing called "God mode" whatsoever.